docs/content/en/open_source/upgrading/2.47.md

```diff
@@ -12,4 +12,15 @@ There are no special instructions for upgrading to 2.47.x. Check the [Release No
 
 ## Removal of Asynchronous Import
 
-Please note that asynchronous import has been removed as it was announced in 2.46. If you haven't migrated from this feature yet, we recommend doing before upgrading to 2.47.0
\ No newline at end of file
+Please note that asynchronous import has been removed as it was announced in 2.46. If you haven't migrated from this feature yet, we recommend doing before upgrading to 2.47.0
+
+## Wizcli Parsers Hashcode Changes
+**Hash Code changes** Wizcli parsers have been updated to populate more fields in release 2.47.3. Some of these fields are part of the hash code calculation. To recalculate the hash code please execute the following command:
```

Comment:
`Wizcli parsers have been updated to populate more fields. Some of these fields are part of the hash code calculation. To recalculate the hash code please execute the commands below. Please note that due to the changes in the parser deduplication might not always happen against findings that were already present in Defect Dojo.`

Reply:
I don't know why but Github doesn't offer me the "Suggest Changes" option here.